### PR TITLE
Fixes to sources registration etc

### PR DESCRIPTION
**Pull request by j00bar**
Fixes to Sources registration and associated bug fixes, plus preparation for role upload.

### CODE REVIEW

File: roles/satellite-receptor/defaults/main.yml [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+install_receptor_from_rpm: yes
 satellite_url: "https://satellite.example.com/"
 satellite_user: "fifi_user"
 satellite_password: "0p3ns3s4m3!"
 satellite_ca_file: "/etc/pki/tls/certs/ca-bundle.crt"
 satellite_validate_certs: yes
 receptor_config_dir: "/etc/receptor"
-# TODO: Find a better place for this, something like /var/lib/receptor
-receptor_data_dir: /tmp
+receptor_data_dir: /var/data/receptor
```
Comment (ares) [inline roles/satellite-receptor/defaults/main.yml]: do we need to create/setup this from rpm?

**Comment by ares** [comment]
nice

File: roles/satellite-receptor/defaults/main.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 receptor_packages:
   - receptor
   - python3-receptor-satellite
 c_rh_c_host: "cert.cloud.redhat.com"
-# TODO: Obtain this dynamically?
-source_type_id: "8"
+skip_satellite_org_id_list:
+  - 1
```
Comment (adamruzicka) [inline roles/satellite-receptor/defaults/main.yml]: Why are we skipping org with id 1 by default? (if I'm reading this right)

Comment (ares) [reply roles/satellite-receptor/defaults/main.yml]: oh, sorry that's my fault of not explaining well, nuc1 setup has forklift manifest there which is problematic, we shoud ignore such error as it's our seutp oddity, we shouldn't be generally skipping org with id 1

Jag if that prevents you from testing, we can probably delete the broken manifest from that org

Comment (ares) [reply roles/satellite-receptor/defaults/main.yml]: unless this was temporary workaround you planned to remove before merge...

**Comment by adamruzicka** [comment]
I just ran it and it was a success. I ran into some minor issues, but here's the fix for them https://github.com/project-receptor/satellite-receptor-installer/pull/5

File: roles/satellite-receptor/defaults/main.yml (hunk 1 above)
Comment (adamruzicka) [reply roles/satellite-receptor/defaults/main.yml]: Currently this role creates it, but having the rpm do that would be nicer

Comment (j00bar) [reply roles/satellite-receptor/defaults/main.yml]: I'm going to make a PR against hte spec file.

File: roles/satellite-receptor/defaults/main.yml (hunk 2 above)
Comment (j00bar) [reply roles/satellite-receptor/defaults/main.yml]: Thank you for the reminder :)

**Pull request merged**